docs/community/intro.md

```diff
@@ -4,12 +4,28 @@ sidebar_label: 🧐 How It Works
 description: Overview of what SourceCred is and how it works
 ---
 
-Everything people do to support a project—like writing code, filing bug reports, organizing meetups, or even mediating hard discussions—is considered a contribution.
+SourceCred is an algorithm that evaluates the value of, and shows the relationships between, the contributions that collaborators make to a project. 
```

Comment:
IMO, the social algorithm is just one part of SourceCred--grain and boosting are key components too. However, I thin kthis rephrase is a good improvement over the content currently here.